src/video_core/renderer_opengl/renderer_opengl.h

```diff
@@ -89,6 +89,13 @@ class RendererOpenGL : public RendererBase {
     void ConfigureFramebufferTexture(TextureInfo& texture,
                                      const GPU::Regs::FramebufferConfig& framebuffer);
     void DrawScreens(const Layout::FramebufferLayout& layout, bool flipped);
+    void ResetSecondLayerOpacity();
+    void ApplySecondLayerOpacity();
+    void DrawBottomScreen(const Layout::FramebufferLayout& layout,
+                          const Common::Rectangle<u32>& bottom_screen,
+                          const bool& stereo_single_screen);
```

Comment:
```suggestion
                          bool stereo_single_screen);
```
ditto for the other function and cpp files as well